### PR TITLE
my domain name and details

### PR DESCRIPTION
**Pull request by keviiixaviiii**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/kevin-xavier.is-a.dev
Comment (Stef-00012) [inline domains/kevin-xavier.is-a.dev:1]: Remove `.is-a.dev` and add `.json`

File: domains/kevin-xavier.is-a.dev [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "subdomain": "kevin-xavier",
```
Comment (Stef-00012) [inline domains/kevin-xavier.is-a.dev:2]: `subdomain` is not a valid key in is-a.dev

File: domains/kevin-xavier.is-a.dev [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "subdomain": "kevin-xavier",
+  "owner": {
+    "email": "iamkevinxavi@gmail.com",
+    "github": "keviiixaviiii"
+  },
+  "records": {
+    "CNAME": "keviiixaviiii.github.io"
```
Comment (Stef-00012) [inline domains/kevin-xavier.is-a.dev:8]: I cannot reach your website, 404 page

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by orxngc** [comment]
Closing as stale.

**Pull request closed without merging**